OSDOCS-4198: Explain optional capabilities (#50830)
openshift/openshift-docs: modules/explanation-of-capabilities.adoc

// Module included in the following assemblies:
//
// *post_installation_configuration/cluster-capabilities.adoc

:_content-type: REFERENCE
[id="explanation_of_capabilities_{context}"]
= Optional cluster capabilities in {product-title} {product-version}

Optional cluster capabilities are components that can be disabled during the installation process or enabled after installation. After a cluster capability is enabled, it cannot be disabled. 

> Currently, cluster Operators provide the features for these optional capabilities. The following summarizes the features provided by each capability and what functionality you lose if it is disabled.

Reviewer: ```suggestion
Currently, cluster Operators provide the features for these optional capabilities. The following summarizes the features provided by each capability and what functionality will not be present if it is disabled.
```
